frontend/__tests__/components/route-pages.spec.tsx

```diff
@@ -196,7 +196,7 @@ describe(RouteStatus.displayName, () => {
     };
 
     const wrapper = mount(<RouteStatus obj={route} />);
-    expect(wrapper.find('.pficon-ok').exists()).toBe(true);
+    expect(wrapper.find('.co-icon-and-text__icon').exists()).toBe(true);
```

Comment:
This might be ok because we do check the text, but do we want to test for the exactly icon we expect?

Reply:
Unfortunately, by default the component doesn't output anything that identifies the icon.  :(  We could add a class for this purpose.
![Screen Shot 2019-07-15 at 4 08 06 PM](https://user-images.githubusercontent.com/895728/61245606-e7b68e80-a71a-11e9-948e-5d10c97ff913.png)